packer/scripts/update.sh

```diff
@@ -39,5 +39,5 @@ apt-get -y update;
 echo "upgrade all installed packages incl. kernel and kernel headers"
 apt-get -y dist-upgrade -o Dpkg::Options::="--force-confnew";
 
-reboot
-
+sudo shutdown -r +1
+sleep 5m
```

Comment:
it will fix the disconnected issue :)

Reply:
Just changed it back to `reboot` as it's much faster and works now... (I tested it for > 10 times)
(maybe because of 
```diff
  provisioner "shell" {
    script            = "./scripts/update_apt_packages.sh"
+    expect_disconnect = true
  }
```